tests/osautoinst/test_running.pm

```diff
@@ -8,8 +8,21 @@ sub run {
     type_string "clear\n";
 }
 
+sub get_log {
```

Comment:
1. there is no `utils.pm` in this test distribution
2. as soon as we want to use it elsewhere it should be moved to a more generic file. I am judging with [no functionality is added early](http://www.extremeprogramming.org/rules/early.html) here